corehq/apps/app_manager/tests/test_views.py

```diff
@@ -306,3 +312,127 @@ def apps_modules_setup(test_case):
         Application.get_db().delete_doc(test_case.linked_app.id)
         Application.get_db().delete_doc(test_case.deleted_app.id)
         Application.get_db().delete_doc(test_case.other_app.id)
+
+
+class TestDownloadCaseSummaryViewByAPIKey(TestCase):
+    """Test that the DownloadCaseSummaryView can be accessed with an API key."""
+
+    @classmethod
+    def setUpClass(cls):
+        # Set up a domain and an app.
+        cls.domain = Domain.get_by_name("test-domain")
+        if not cls.domain:
+            cls.domain = Domain(name="test-domain", is_active=True)
+        cls.domain.save()
+        cls.app = Application.new_app("test-domain", "TestApp")
+        cls.app.save()
+
+        # Set up the cls.web_user, and give the cls.web_user access to the cls.domain.
```

Comment:
`WebUser` has a `create` method that takes care of setting up the django and couch users, setting a password, etc.

Reply:
(for clarity: this is not blocking feedback)